### PR TITLE
Don't recommend Debian repos anymore

### PR DESCRIPTION
**Pull request by rubenwardy**
Alternatively, if Debian has something similar to a PPA then that would be good to mention there

image

### CODE REVIEW

**Review by Calinou — APPROVED** [review]

**Comment by Desour** [comment]
"the repos are outdated" could also be added to ubuntu, to drive away users from using the ubuntu repos version instead of ppa or flatpack.

File: downloads.html [hunk 1]
```diff
@@ -99,7 +99,11 @@ <h3>Packages</h3>
               <a href="https://launchpad.net/~minetestdevs/+archive/ubuntu/daily-builds">Daily PPA</a>
           </li>
           <li>Debian -
-              <a href="https://packages.debian.org/search?keywords=minetest">Stable</a></li>
+              use
+              <a href="https://flathub.org/apps/details/net.minetest.Minetest">Flatpak</a>
+              or compile from
+              <a href="https://github.com/minetest/minetest/tree/stable-5">source</a>,
```
Comment (sfan5) [inline downloads.html:105]: rather big usability issue: if you read the docs in the stable version of the repo the docs give you the git commands to download from master. it's not even mentioned that that's not the version you might want

Comment (rubenwardy) [reply downloads.html:105]: This is already an issue with the "source" section of the downloads page

Comment (sfan5) [reply downloads.html]: Sure but maybe we shouldn't so prominently be linking it then. Remove the `<a>` from "source"?

**Review by sfan5 — APPROVED** [review]

**Pull request merged**